public/app/features/integrated-alerting/components/AlertRules/AddAlertRuleModal/AddAlertRuleModal.types.ts

```diff
@@ -1,8 +1,10 @@
 import { SelectableValue } from '@grafana/data';
+import { AlertRule } from '../AlertRules.types';
 
 export interface AddAlertRuleModalProps {
   isVisible: boolean;
   setVisible: (value: boolean) => void;
+  alertRule: AlertRule;
```

Comment:
```suggestion
  alertRule?: AlertRule;
```
To not force to pass undefined.